app/services/audit/AuditService.scala

```diff
@@ -98,5 +98,5 @@ object AuditTypes extends Enumeration {
 }
 object EventData extends Enumeration {
   type Data = Value
-  val EORI, LRN, MRN, DUCR, DecType, ChangeReason, ChangeDescription, FullName, JobRole, Email, Confirmed, SubmissionResult, Success, Failure = Value
+  val eori, lrn, mrn, ducr, decType, changeReason, changeDescription, fullName, jobRole, email, confirmed, submissionResult, Success, Failure = Value
```

Comment:
The same question like in movements, what about Success and Failure?

Reply:
I think they get used as a value, rather than a key in the JSON

Reply:
That is a value not a key. TxM wanted the keys to be `camelCase`